### PR TITLE
Fix ruby 'last argument as keyword parameters' deprecation

### PR DESCRIPTION
**Pull request by Lomey**

### CODE REVIEW

File: lib/active_support/cache/redis_store.rb [hunk 1]
```diff
@@ -82,7 +82,7 @@ def write(name, value, options = nil)
           if options[:expires_in].present? && options[:race_condition_ttl].present? && options[:raw].blank?
             options[:expires_in] = options[:expires_in].to_f + options[:race_condition_ttl].to_f
           end
-          entry = options[:raw].present? ? value : Entry.new(value, options)
+          entry = options[:raw].present? ? value : Entry.new(value, **options)
```
Comment (tubbo) [inline lib/active_support/cache/redis_store.rb:85]:
```suggestion
          entry = options[:raw].present? ? value : Entry.new(value, **options.to_h.symbolize_keys)
```

**Comment by tubbo** [comment]
not sure why all those tests failed, probably due to the tests not having been run in a while so i didn't notice any of the deprecations

**Comment by udicohen** [comment]
Is there any way to make progress on this? It is a simple fix and saves a lot warning messages 🙏

**Comment by Lomey** [comment]
@tubbo, okay, sure

**Comment by Lomey** [comment]
Tests were failed because of this commit https://github.com/mperham/connection_pool/commit/01ca704af1078b9105fddecd36448ec8c62035d5#diff-d19d56f7b52363b0a7d63a328d77e0fdR124 in connection_pool gem in v2.2.3, so as a temporary fix I changed version to 2.2.2

btw, if add a 'global scope' for `Process::CLOCK_MONOTONIC` constant `::Process::CLOCK_MONOTONIC` all will be passed

**Comment by tubbo** [comment]
@Lomey I see. Well, now that you got the build passing I will merge this PR, but I'm going to restore the original version constraint of `connection_pool` and just patch that method for now.

**Review by tubbo — APPROVED** [review]

File: redis-activesupport.gemspec [hunk 2]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'bundler'
   s.add_development_dependency 'mocha',    '~> 0.14.0'
   s.add_development_dependency 'minitest', '>= 4.2', '< 6'
-  s.add_development_dependency 'connection_pool', '~> 2.2.0'
+  s.add_development_dependency 'connection_pool', '= 2.2.2'
```
Comment (PikachuEXE) [inline redis-activesupport.gemspec:28]: Any reason to use this instead of `~> 2.2.2`?

**Pull request merged**